recipes/plugins_example_document.ipynb

```diff
@@ -0,0 +1,329 @@
+    "\n",
+    "kili = Kili(api_key=main_api_key, api_endpoint=api_endpoint)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7063407a",
+   "metadata": {},
+   "source": [
+    "## Step 2 : Create the project\n",
+    "\n",
+    "First, we need to create a new project. For our example, it will be a project of type `IMAGE` and it will have the following `jsonInterace` :"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "cc56bc57",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create the project\n",
+    "\n",
+    "title = \"Demo QA Bot PDF\"\n",
+    "description = \"Demo QA Bot PDF\"\n",
+    "input_type = \"IMAGE\"\n",
+    "\n",
+    "\n",
+    "filename = \"./datasets/document_plugin_data/jsoninterface.json\"\n",
+    "\n",
+    "with open(filename, \"r\") as f:\n",
+    "    json_interface = json.load(f)\n",
+    "\n",
+    "project = kili.create_project(\n",
+    "    title=title, description=description, input_type=input_type, json_interface=json_interface\n",
+    ")\n",
+    "\n",
+    "project_id = project[\"id\"]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "b4ef53d4",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "100%|██████████| 1/1 [00:00<00:00,  1.17it/s]\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "{'id': 'cldbms12c00er0jwc7iiy21dq', 'metadataTypes': {'accuracy': 'number'}}"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Add assets\n",
+    "\n",
+    "path = \"./datasets/document_plugin_data/\"\n",
+    "\n",
+    "content_array = glob.glob(os.path.join(path, \"*.png\"))\n",
+    "names_array = [os.path.basename(f) for f in content_array]\n",
+    "\n",
+    "\n",
+    "project = kili.append_many_to_dataset(\n",
+    "    project_id=project_id, content_array=content_array, external_id_array=names_array\n",
+    ")\n",
+    "\n",
+    "\n",
+    "kili.update_properties_in_project(\n",
+    "    project_id=project[\"id\"],\n",
+    "    metadata_types={\n",
+    "        \"accuracy\": \"number\",\n",
+    "    },\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4d8490f2",
+   "metadata": {},
+   "source": [
+    "This project has two jobs with several tasks of transcription associated. We will be interested in the *Payment information* job, and more precisely the *IBAN* and the *Currency* sub-jobs. With our plugin, we want to be sure that the labelers write the correct values for the two transcriptions (so we are interested only in the `on_submit` handler of the plugin), since we know for sure that the IBAN should start with the 2 letters *FR*, and that the currency should be one of : *EUR*, *USD*. At then end, we also calculate an accuracy and we insert it in the `json_metadata` of the asset.\n",
```

Comment:
In the actual plugin it's EURO and DOLLAR, not EUR and USD

Reply:
same here, thanks for the thorough review gheorghe !